gramps/plugins/graph/gvhourglass.py

```diff
@@ -456,7 +456,7 @@ def add_menu_options(self, menu):
             _("Show Sosa / Sosa-Stradonitz / Ahnentafel number under all others informations."))
         menu.add_option(category_name, "ahnentafelnum", ahnentafelnumvisible)
 
-        datesymbol = BooleanOption(_("Add Symbol for date"), False) # 10826
+        datesymbol = BooleanOption(_("Show symbol for events"), False) # 10826
         datesymbol.set_help(
-            _("Add a symbol for date. o for birth, + for decease and x for wedding."))
+            _("Add a symbol for date. o for birth, + for deceased and x for marriage."))
```

Comment:
Missed one:   

*  Add a symbol for date. > Show symbol for events.

Reply:
sure?

Reply:
Yes @wroldwiedbwe English phrasing is correct ; as you are not adding a date on showing a symbol beside the three event types.  

@khrys63 Haven't tested, but hopefully the symbols don't show if the events don't exist or are blank?

Reply:
with last commit it's good :)

![image](https://user-images.githubusercontent.com/21293925/154119943-ae524559-603a-4763-b6b8-936e927d0136.png)